overlord/snapstate/snapstate_test.go

```diff
@@ -15466,6 +15470,23 @@ func (s *snapmgrTestSuite) TestInstallUserDaemonsChecksFeatureFlag(c *C) {
 	c.Assert(err, ErrorMatches, "experimental feature disabled - test it by setting 'experimental.user-daemons' to true")
 }
 
+func (s *snapmgrTestSuite) TestInstallUserDaemonsUsupportedOnTrusty(c *C) {
```

Comment:
Install-related tests have been moved to snapstate_install_test.go (therefore a conflict here), please move new tests there.